.github/workflows/ci.yml

```diff
@@ -227,12 +227,12 @@ jobs:
       - run: yarn start-postgres-${{ matrix.postgres-version }}
       - name: Integration Tests
         run: yarn lerna run test-integration --scope=@sequelize/core
-  test-oldest-latest:
+  test-latest:
```

Comment:
This is not really what I had in mind. I was thinking that we just remove mssql from the dialects in test-oldest-latest and create two new jobs; test-mssql-oldest and test-mssql-latest

Reply:
Ok, sure. Updated now